atomic_reactor/utils/cachi2.py

```diff
@@ -287,4 +292,89 @@ def clone_only(remote_source: Dict[str, Any]) -> bool:
     if pkg_managers is not None and len(pkg_managers) == 0:
         return True
 
+    # only git-submodule
+    if pkg_managers is not None and pkg_managers == ['git-submodule']:
+        return True
+
     return False
+
+
+def has_git_submodule_manager(remote_source: Dict[str, Any]) -> bool:
+    """Returns true when for specific remote source git-submodule manager is requested"""
+    pkg_managers = remote_source.get("pkg_managers") or []
```

Comment:
Why not

```suggestion
    pkg_managers = remote_source.get("pkg_managers", [])
```

Not required, just to satisfy my curiosity...

Reply:
in case that somebody specify `pkg_managers:`  as empty in yaml a.k.a `null`, it would explode later. Probably jsonschema should catch it, but defensive programming :D

Reply:
Isn't that what the second arg to `get` is for? Maybe I'm misreading it...